Add solution in solutions.md

DLemming opened the pull request:
RegEx to find matching phone numbers.

Works correctly on kaputt.txt.

Commit:
Add solution in solutions.md

jo3rn commented:
Vielen Dank für deine Lösung, @DLemming !

Leider fehlt eine Kleinigkeit zur vollständigen Erkennung. Diese Muster werden nicht gematcht:
```text
+XX XX XXXXXXX
+XX-XX-XXXXXXX
```

Deshalb findet man momentan 11 von 13 Nummern. In `kaputt.txt` sind das diese zwei noch fehlenden Nummern:
```text
+49 75 7725037
+49-05-1476601
```

DLemming commented:
So jetzt hat. Sollte klappen!

Zwar nicht kürzer als die Musterlösung, aber der einzige PR dazu :D
(Vielleicht doch nen kleinen Preis? :o)

DLemming committed:
Update solutions.md

Jetzt hat. Sollte funktionieren.

Leider nicht kürzer als die Musterlösung, aber die einzige :P

jo3rn committed:
Merge branch 'main' into solution

jo3rn commented:
ich nehm die Lösung auf jeden Fall mit auf für die Bonuspunkte. für einen kleinen Preis gibt es in der allerletzten SU noch Möglichkeiten 🥇
